web/src/components/SubscriptionFormHTML.jsx

```diff
@@ -0,0 +1,26 @@
+import { Button, Input, Space, Flex } from "antd";
+
+const SubscriptionFormHTML = () => {
+	return (
+		<Flex justify="center" align="center" style={{ height: "100vh" }}>
```

Comment:
Just one small thing. The styling to position the component will happen in the page ticket. This component is just about creating the component, not positioning it on the page.